ceph/ceph_admin/manager.py

```diff
@@ -0,0 +1,93 @@
+"""Perform the manager operations via Ceph's cephadm CLI."""
+from json import loads
+
+from .ceph import CephCLI
+
+
+class Manager(CephCLI):
+    """Manager interface for ceph mgr <action>"""
+
+    SERVICE_NAME = "manager"
```

Comment:
We had this variable defined for `orch appy`... IMO, we could avoid this variable.

Reply:
Done